site/en/docs/extensions/mv3/manifest/author/index.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Manifest - Author"
+date: 2022-10-03
+updated: 2022-10-03
+description: Reference documentation for the author name property of manifest.json.
+---
+## Manifest - Author
```

Comment:
I should have caught this before. (I'm still getting familiar with our content.) Why is this heading here? It just duplicates the title.

Reply:
Thank you, missed that this was how it worked as well. Corrected!